### ISSUE TITLE
Should `vat` select JSX tags?

### ISSUE THREAD

**Issue by skube** [VSCodeVim/Vim#2431]
**BUG REPORT**:

**What happened**:
Using JSX with custom Components and using `vit` or `vat`, custom JSX tags are ignored.

**What did you expect to happen**:
I expect custom JSX html-like tag to be selected

**How to reproduce it (as minimally and precisely as possible)**:
```
<div>
  <MyCustomComponent>
    |
  </MyCustomComponent>
</div>
```

If cursor is at pipe (|), then hitting `vat`selects the outer `div` and seems to ignore the `<MyCustomComponent>` tag. 

**Environment**:

- Extension (VsCodeVim) version: 0.11.1
- VSCode version: 1.21.0 
- OS version: 10.13.3

**Comment by Chillee** [comment]
Could you provide a repro file? I can't reproduce this on a new file.

**Comment by skube** [comment]
Actually, I've isolated the issue. It's when the prop contains an element that vim gets confused.

```
<div>
  <Component
    prop={<div>lorem</div>}
  >
    | 
  </Component
</div>
```

**Comment by xmbhasin** [comment]
I believe this was fixed in #2921. The second example provided is missing a `>` in the second `Component` tag, after fixing that `vat` seems to work as expected.
